course7/homeworks/kushagra/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(my_ls)
+
+find_package(Boost 1.55 REQUIRED COMPONENTS filesystem system)
+
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fexceptions -Wextra -std=c++14")
```

Comment:
Don't set too many compiler-specific flags, and definitely never set -g. Leave it to the person building your project to decide whether it should be a release or debug build.